src/com/carolinarollergirls/scoreboard/Stats.java

```diff
@@ -0,0 +1,75 @@
+package com.carolinarollergirls.scoreboard;
+/**
+ * Copyright (C) 2008-2012 Mr Temper <MrTemper@CarolinaRollergirls.com>
+ *
+ * This file is part of the Carolina Rollergirls (CRG) ScoreBoard.
+ * The CRG ScoreBoard is licensed under either the GNU General Public
+ * License version 3 (or later), or the Apache License 2.0, at your option.
+ * See the file COPYING for details.
+ */
+
+import java.util.List;
+
+import com.carolinarollergirls.scoreboard.event.ScoreBoardEventProvider;
+
+public interface Stats extends ScoreBoardEventProvider
+{
+	public ScoreBoard getScoreBoard();
+
+	public List<PeriodStats> getPeriodStats();
+
+	public static interface PeriodStats extends ScoreBoardEventProvider {
+    public int getPeriodNumber();
```

Comment:
can you fix the indentation in this file?

Reply:
Done